### ISSUE TITLE
[BUG] v1.3.0 published with invalid Gradle Module Metadata

### ISSUE THREAD

**Issue by msfjarvis** [slackhq/EitherNet#58]
### Describe the bug
The GMM for v1.3.0 contains invalid capabilities entries for the testFixtures variants so Gradle fails to resolve the dependency even if you are not relying on the test fixtures.

Capabilities in v1.2.1

<details>
<summary>Details</summary>

```json5
// curl -sL https://repo.maven.apache.org/maven2/com/slack/eithernet/eithernet/1.2.1/eithernet-1.2.1.module | jq '.variants[].capabilities? | select( . != null )'
[
  {
    "group": "com.slack.eithernet",
    "name": "eithernet-test-fixtures",
    "version": "1.2.1"
  }
]
[
  {
    "group": "com.slack.eithernet",
    "name": "eithernet-test-fixtures",
    "version": "1.2.1"
  }
]
```

</details>

Capabilities in v1.3.0

<details>
<summary>Details</summary>

```json5
// curl -sL https://repo.maven.apache.org/maven2/com/slack/eithernet/eithernet/1.3.0/eithernet-1.3.0.module | jq '.variants[].capabilities? | select( . != null )'
[
  {
    "group": "",
    "name": "eithernet-test-fixtures",
    "version": "unspecified"
  }
]
[
  {
    "group": "",
    "name": "eithernet-test-fixtures",
    "version": "unspecified"
  }
]
[
  {
    "group": "",
    "name": "eithernet-test-fixtures",
    "version": "unspecified"
  }
]
```

</details>

Build failure log

<details>
<summary>Details</summary>

```plaintext
* What went wrong:
Execution failed for task ':api:compileDebugKotlin'.
> Error while evaluating property 'filteredArgumentsMap' of task ':api:compileDebugKotlin'.
   > Could not resolve all files for configuration ':api:debugCompileClasspath'.
      > Could not resolve com.slack.eithernet:eithernet:1.3.0.
        Required by:
            project :api
         > Could not resolve com.slack.eithernet:eithernet:1.3.0.
            > Could not parse module metadata https://repo.maven.apache.org/maven2/com/slack/eithernet/eithernet/1.3.0/eithernet-1.3.0.module
               > missing 'group' at /variants[2]/capabilities[0]
```

</details>

### Requirements
* [x] I've read and understood the [Contributing guidelines](../CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### To Reproduce

Add the EitherNet 1.3.0 dependency to a project and attempt to build it

### Expected behavior

Gradle resolves the dependency and builds the project

#### Reproducible in:

Project version: 1.3.0

#### Additional context

I noticed this in a regular Renovate PR to [my app](https://github.com/msfjarvis/compose-lobsters/pull/357) so you can repro it there but it has a rather massive dependency tree so it's probably easier to use a fresh skeleton project .

**Comment by ZacSweers** [comment]
Thanks for the report, I'll take a look soon

**Comment by ZacSweers** [comment]
Fixed in 1.3.1!
